scripts/modules/elastic_stack.py

```diff
@@ -725,6 +725,8 @@ def __init__(self, **options):
             self.environment["XPACK_MONITORING_ENABLED"] = "true"
             if self.at_least_version("6.3"):
                 self.environment["XPACK_XPACK_MAIN_TELEMETRY_ENABLED"] = "false"
+            if self.at_least_version("7.6"):
```

Comment:
I'd say it's `7.5`.
This is the version where we added the _opted-in_ by default and I've checked in the `kibana-docker` file and this property exists for that version